docs/introduction/diffusion_intro/diffusion_intro.rst

```diff
@@ -0,0 +1,142 @@
+
+Mean Squared Displacement analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A common method for determining the diffusion coefficient, is to first compute the mean squared displacement (MSD) as a function of lag time for each track. 
+For purely diffusive motion, the first part of the MSD curve is linear, with the slope proportional to the diffusion coefficient. For 1D diffusion the MSD is given by: :math:`MSD = 2D\Delta t`.
+The diffusion coefficient can be obtained by fitting a linear function to the MSD curve, as illustrated in the figure below (simulated data):
+
+.. image:: figures/MSD_analysis.png
+
+Note that for the MSD curve, the 'lag time' (aka 'time interval') is given on the x-axis, rather than 'time'. 
+To compute the MSD for a certain lag time, you scan over a track, one data point at a time and compute the displacement squared 
+from that time until that time + the lag time. So each point on the MSD curve, is an average of displacements for a specific lag time  (see page ... for the equation of MSD). 
+
+The largest possible lag time is the duration of a track and the shortest possible lag is the time resolution, the line time of the kymograph. 
+
+Note that the image above does not show the complete MSD curve, but only up to a lag time of 5 seconds. 
+The full MSD curve looks as follows:
+
+.. image:: figures/MSD_full.png
+
+As you can see, the full MSD curve becomes less and less linear for larger lags times. The funky shape of the MSD curve for large lag times is caused by:
+
+#. The MSD at shorter time lags is averaged over more lags and therefore more accurate
+
+#. Consecutive data points in the MSD are highly correlated
+
+The MSD curve for larger lags is not suitable for a linear fit. An important step in MSD analysis, is therefore to determine which number of lags is optimal for a linear fit and obtaining the diffusion coefficient.
+Taking too many lags, may result in an MSD curve that is not fully linear, while taking too little lags may not give enough data points for a good estimate of the diffusion coefficient.
+There are various methods for determining the optimal number of lags. The method recommended in Pylake automatically determines the optimal number of lags for you :cite:`michalet2012optimal`(named 'OLS' in Pylake). 
+
+Covariance Based Estimator
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Covariance Based Estimator (:ref:`'CVE' <diffusion_cve>`) in Pylake is a method for determining the diffusion coefficient that does not rely on the MSD :cite:`vestergaard2014optimal,vestergaard2016optimizing`. 
+(CITE STIGLER) This method directly computes the diffusion coefficient from the displacements.
+
+Comparing CVE and MSD analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since there is no fitting involved, CVE is fast and unbiased (unbiased means that the more data you get, the closer the average gets to the real diffusion constant). 
```

Comment:
Yeah, I think it's better indeed. For these things, it would be really awesome if we had some sort of glossary functionality on ReadTheDocs. Where you hover over a word and see a small glossary item with an image or something.

Many of these things are so easy to show in a plot, but so hard to put into words.